.github/pull_request_template.md

```diff
@@ -1,6 +1,8 @@
 Thank you for contributing to Quire! Please complete the form below to submit your pull request for review. 
 
-*For the Title of this pull request, please use the format "Type/Issue-#: Brief description." For Type, the options are Fix, Feature, Docs, or Chore. Issue-# is only needed if this pull request addresses an [exisiting issue](https://github.com/thegetty/quire/issues).*
+*For the Title of this pull request, please use the format "Type/Issue-#: Brief description." For Type, the options are Fix, Feature, Docs, or Chore. Issue-# is only needed if this pull request addresses an [existing issue](https://github.com/thegetty/quire/issues).*
+
+**Before submitting your PR, make sure that you can run the following commands: `quire preview`, `quire build`, `quire pdf`, and `quire epub`. If any of those core quire commands throw errors/yield unexpected behavior, we will NOT review the PR!**
```

Comment:
@cbutcosk let's pull this out and discuss with @Erin-Cecele updating the pull-request template separately.